hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -1110,8 +1108,7 @@ public void update(HoodieRollbackMetadata rollbackMetadata, String instantTime)
       HoodieInstant deltaCommitInstant = new HoodieInstant(false, HoodieTimeline.DELTA_COMMIT_ACTION, commitToRollbackInstantTime);
       if (deltacommitsSinceCompaction.containsInstant(deltaCommitInstant)) {
         LOG.info("Rolling back MDT deltacommit " + commitToRollbackInstantTime);
-        String rollbackInstantTime = createRollbackTimestamp(instantTime);
-        if (!getWriteClient().rollback(commitToRollbackInstantTime, rollbackInstantTime)) {
+        if (!getWriteClient().rollback(commitToRollbackInstantTime, instantTime)) {
```

Comment:
we need per log file marker fix. even if not in entirely, there is some gap here. 
even if commit to rollback was never applied to MDT, we need to apply rollbacks to track log files created by original commit of interest. 
lets create a ticket to track the pending work.

Reply:
Did you check the MDT rollback hanling, we have some logic for log files rollback for some scenarios like the RLI.